chore: generate capacitor 5 plugin docs (#159)
ionic-team/capacitor-docs: docs/apis/screen-orientation.md

---
title: Screen Orientation Capacitor Plugin API
description: The Screen Orientation API provides methods to lock and unlock the screen orientation.
editUrl: https://github.com/ionic-team/capacitor-plugins/blob/main/screen-orientation/README.md
editApiUrl: https://github.com/ionic-team/capacitor-plugins/blob/main/screen-orientation/src/definitions.ts
sidebar_label: Screen Orientation
---

# @capacitor/screen-orientation

The Screen Orientation API provides information and functionality related to the orientation of the screen.

## Install

```bash
npm install @capacitor/screen-orientation
npx cap sync
```

## iOS

Locking the Screen Orientation only works for the Capacitor View Controller only, but not other View Controllers being presented (such as the one presented by Browser plugin).

> For also lock presented View Controllers, this code can be added to the app's `AppDelegate.swift` file:

Reviewer: ```suggestion
To lock all presented View Controllers, this code can be added to the app's `AppDelegate.swift` file:
```
